src/Messages/OWSMessageSender.m

```diff
@@ -17,6 +17,7 @@
 #import "SignalRecipient.h"
 #import "TSAccountManager.h"
 #import "TSAttachmentStream.h"
+#import "TSBlockingManager.h"
```

Comment:
nit: I prefer the 3-letter class prefix (OWS) in line with apples style guide, to avoid the (admittedly small chance) that we're one day clobbered by a new apple framework.